Makefile

```diff
@@ -1,3 +1,15 @@
+pip:
```

Comment:
is `install` a better target?
users can run `make install`
instead of `make pip`